src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -123,19 +127,23 @@
 
     private final PFFController<Vector2> tipController;
 
-    public DrivebaseSubsystem(SwerveModule fl, SwerveModule fr, SwerveModule bl, SwerveModule br, Gyroscope g,
-            double moduleMaxVelocityMetersPerSec) {
+    public DrivebaseSubsystem() {
+        var comp = Robot.getInstance().isCompetition() && !Robot.isSimulation();
```

Comment:
Simulation or not should only affect whether the absolute encoder is used, not the gyro selection